packages/web-config-server/src/apiV1/dataBuilders/generic/dataDownload/rawDataValues.js

```diff
@@ -22,6 +22,15 @@ const expandSurveyCodes = surveys => {
   );
 };
 
+const convertValueByType = (type, value) => {
+  switch (type) {
+    case 'Binary':
+    case 'Checkbox':
+      return value === 1 ? 'Yes' : 'No';
+    default:
+      return value;
+  }
+}
```

Comment:
Convert `1/0` to `Yes/No` because we actually want to show `Yes/No` values for Binary in custom export

Reply:
Would prefer to do this in a different way. I think the way we'd do it with DHIS2 at the moment is to look at the data element metadata, which can include an `options` field with each. Do a find on `includeOptions` as a starting point. Not saying this is perfect - but we'd be best to keep consistent where we can

Reply:
Thanks Edwin for pointing that out, I agree that we should keep it consistent!

Just checking if I'm in the right direction. So after having a look at how dhis2 does `includeOptions`, it returns something like below when we do fetchDataElements:
```
[
   {
     options: {
       Church: 'Church',
       Workplace: 'Workplace',
       School: 'School',
       Community: 'Community'
     },
     code: 'HP31n',
     name: 'HP NCD Risk Factor Screening: Setting Type',
     id: 'fvmaQAPl5bI'
   }
]
```

I think we can do the same and it can return something like (with actual value keyed by sanitized value) :
```
[
   {
     options: {
       1: 'Yes',
       0: 'No',
     },
     code: 'ABC',
     ...
   }
]
```

Reply:
Yes exactly. This will probably also result in the Laos ones becoming Laos text...but maybe that's a good thing!

Reply:
> With that, binary questions with no labels will also have to have their options defined as {Yes,No}

You mean by the code rather than in the database right? Like we can default to that in the data fetching logic?